xlson/app.py

```diff
@@ -0,0 +1,37 @@
+""" Importing required strings """
+from typing import Any
+
+from django import forms
+from django.http import HttpRequest, HttpResponse
+from django.shortcuts import render
+from django_micro import configure, run
+
+DEBUG = True
+configure(locals())
+
+
+class UploadFileForm(forms.Form):
```

Comment:
The file validation needs to go into the form class.  Look at https://djangosnippets.org/snippets/1303/ and https://medium.com/@literallywords/server-side-file-extension-validation-in-django-2-1-b8c8bc3245a0 for inspiration.

We want to:

- Validate that it is the correct content type (XLS file basically).  The allowed content types need to be configurable via settings i.e. just above line 10 in this file
- Not above a certain maximum file size (lets say 10MB for now).  This max file size needs to be configurable via settings i.e. just above line 10 in this file